cmake/kokkos_arch.cmake

```diff
@@ -346,6 +431,9 @@ CHECK_CUDA_ARCH(VOLTA70  sm_70)
 CHECK_CUDA_ARCH(VOLTA72  sm_72)
 CHECK_CUDA_ARCH(TURING75  sm_75)
 
+IF (KOKKOS_ENABLE_CUDA)
+  SET(KOKKOS_COMPILER_CUDA_VERSION "${KOKKOS_VERSION_MAJOR}${KOKKOS_VERSION_MINOR}")
```

Comment:
That's not a very descriptive name.  What do we need that for?

Reply:
#cmakedefine KOKKOS_COMPILER_CUDA_VERSION @KOKKOS_COMPILER_CUDA_VERSION@

Reply:
`#cmakedefine KOKKOS_COMPILER_CUDA_VERSION @KOKKOS_COMPILER_CUDA_VERSION@`

Reply:
I didn't even pick this name. This comes from the `Makefile.kokkos`

Reply:
We have this in for now, and when we have parity with the Makefile.kokkos stuff we clean some stuff out again.